src/meson.build

```diff
@@ -0,0 +1,30 @@
+makeguids_sources = ['makeguids.c', 'util.c']
+makeguids = executable('makeguids', makeguids_sources, c_args : ['-DEFIVAR_BUILD_ENVIRONMENT'])
+
+gen_guids = custom_target('gen-guids',
+                          input : ['guids.txt'],
+                          output : ['guid-symbols.c', 'efivar-guids.h'],
+                          command : [makeguids, '@INPUT@', '@OUTPUT0@', '@OUTPUT1@'])
+
+
+lib_efivar_sources = ['crc32.c', 'dp.c', 'dp-acpi.c', 'dp-hw.c', 'dp-media.c', 'dp-message.c', 'efivarfs.c', 'error.c', 'export.c', 'guid.c', 'lib.c', 'vars.c', 'time.c']
+lib_efivar = library('efivar', lib_efivar_sources, gen_guids, version : '1.@0@'.format(meson.project_version()), soversion : '1')
```

Comment:
soversion defaults to the first component of version, so in many cases you don't need both.